core/docs/config.inc.tpl

```diff
@@ -4,12 +4,12 @@
  */
 $database_type = '{database_type}';
 $database_server = '{database_server}';
-$database_user = '{database_user}';
-$database_password = '{database_password}';
 $database_connection_charset = '{database_connection_charset}';
-$dbase = '{dbase}';
 $table_prefix = '{table_prefix}';
-$database_dsn = '{database_dsn}';
+$dbase = '{dbase}';

```

Comment:
```suggestion
$database_name = '{database_name}';
```

For consistency. Placeholder probably need to left the same, but still.

Reply:
@alroniks will this work when you upgrade MODX?

Reply:
I am not sure, I review it in detail a bit later.